includes/Fields/class-field-twitter.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1); 
```

Comment:
for the standard we are following in this project, we are not going to use `declare(strict_types=1);`